### DISCUSSION TITLE
msys2 package

### DISCUSSION THREAD

**Discussion by Kreijstal** [projectM-visualizer/projectm#848] [Ideas]
I checked, and this project let's itself be compiled with msys2 no problem, would anyone be interested in a msys2 package of projectm?

**Comment by revmischa** [comment]
Don't know what that is but sure if you think it's useful

**Reply by Kreijstal** [reply]
it's a mingw distro for windows, you compile using gcc/mingw instead of propietary msvc

**Comment by kblaschke** [comment]
Not sure how popular msys2 is (also never heard of it until now, as I've never used MinGW outside of building Windows binaries on Linux), but I'd say if it's easy to accomplish and may help to make projectM available to a larger user base, sure, go ahead!

**Comment by C0rn3j** [comment]
I've asked the msys2 folk for a package and they were kind enough to provide it - https://github.com/msys2/MINGW-packages/issues/26889

**Reply by Kreijstal** [reply]
ah didn't know about tauon pretty cool
